website/views.py

```diff
@@ -88,6 +89,11 @@ def serialize_node_summary(node, auth, primary=True, show_path=False):
             'api_url': node.api_url,
             'title': node.title,
             'category': node.category,
+            'isPreprint': bool(node.preprint_file and node.preprint_file._id),
+            'childExists': bool(node.nodes_active),
+            'is_admin': node.has_permission(user, permissions.ADMIN),
+            'is_contributor': node.is_contributor(user),
+            'logged_in': auth.logged_in,
```

Comment:
The data in contextVars.currentUser pertains to the project not its components, reason why I added these to the summary of a component. That way we can get the user's permissions on the component.